### ISSUE TITLE
[BUG] When oidc_enable: false in ansible full.yml, web portal throws Internal Server Error on login

### ISSUE THREAD

**Issue by fj-morales** [UtrechtUniversity/yoda-ansible#226]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When installing YODA via ansible and the oidc_active variable is set to false, the flask.cfg lacks any OIDC configuration, provoking that the file /var/www/yoda/user/user.py fails when trying to access the oidc_domains, which is None; therefore, producing an Internal Server Error

```
File "/var/www/yoda/user/user.py", line 433, in get_login_placeholder
	if len(oidc_domains) == 0 or oidc_domains[0] == "":
		TypeError: object of type 'NoneType' has no len()
````

![image](https://user-images.githubusercontent.com/8096489/221830924-dd41162a-9ffc-4636-8e0e-f82f70112543.png)

### Expected Behavior

When the oidc_active variable is set to false, the YODA login page should show the login prompt to enter the username 

### Steps To Reproduce

Install YODA via ansible with:
 oidc_active: false

### Environment

```markdown
- Yoda: v1.8.6
- Ansible: 2.13.7
- Operating System: Centos 7
- Browser: Chrome
```

**Comment by stsnel** [comment]
Thanks for the bug report, we're tracking this issue as YDA-5151 in our internal bug database.

**Comment by stsnel** [comment]
Fixed for 1.9.0 with https://github.com/UtrechtUniversity/yoda-portal/pull/188 ; also to be cherry-picked to 1.8.7
